packages/fxa-auth-server/scripts/bulk-mailer/normalize-user-records.js

```diff
@@ -13,37 +14,25 @@ module.exports = class UserRecordNormalizer {
    * the a format suitable for the user.
    *
    * @param {Object[]} userRecords
-   * @param {Object} translator
    */
-  normalize(userRecords, translator) {
+  normalize(userRecords) {
     return (
       userRecords
         // no email can be sent if the record does not contain an email
         .filter((record) => !!record.primaryEmail)
-        .map((userRecord) => this.normalizeUserRecord(userRecord, translator))
+        .map((userRecord) => this.normalizeUserRecord(userRecord))
     );
   }
 
-  normalizeUserRecord(userRecord, translator) {
-    this.normalizeAcceptLanguage(userRecord);
-    this.normalizeLanguage(userRecord, translator);
+  normalizeUserRecord(userRecord) {
+    this.normalizeLanguage(userRecord);
     this.normalizeLocations(userRecord);
 
     return userRecord;
   }
 
-  normalizeAcceptLanguage(userRecord) {
-    // The Chinese translations were handed to us as "zh" w/o a country
-    // specified. We put these translations into "zh-cn", use "zh-cn" for
-    // Taiwan as well.
```

Comment:
I ran this by Peiying as well.

I'm not sure if we just didn't support `zh-tw` for a while or if there was a bug, but it's available in the repo now and `zh-tw` users should see `zh-tw` as expected. 🤷‍♀️